src/spyglass/utils/nwb_helper_fn.py

```diff
@@ -158,6 +158,28 @@ def get_data_interface(nwbfile, data_interface_name, data_interface_class=None):
     return None
 
 
+def get_position_obj(nwbfile):
+    """Return the Position object from the behavior processing module.
+    Meant to find position spatial series that are not found by
+    `get_data_interface(nwbfile, 'position', pynwb.behavior.Position)`.
+    The code returns the first `pynwb.behavior.Position` object (technically
+    there should only be one).
+
+    Parameters
+    ----------
+    nwbfile : pynwb.NWBFile
+        The NWB file object.
+
+    Returns
+    -------
+    pynwb.behavior.Position object
+    """
+    for obj in nwbfile.processing["behavior"].data_interfaces.values():
+        if isinstance(obj, pynwb.behavior.Position):
+            return obj
+    return None
```

Comment:
I think I would prefer an explicit error state if the data violates the 'only one' assumption. As is, it would silently return the first. Feel free to adjust to a `logger.warning` if it's reasonable to just return the first

```suggestion
    ret = []
    for obj in nwbfile.processing["behavior"].data_interfaces.values():
        if isinstance(obj, pynwb.behavior.Position):
            ret.append(obj)
    if len(ret) > 1: 
        raise ValueError(f"Found more than one position object in {nwbfile}")
    return ret[0] if ret and len(ret) else None
```

Reply:
I think another approach would be to give the old `get_data_interface` a mapping, like a private dict, so we keep the logic centralized, rather than dev a new func for each data type when nwb is sometimes a moving target anyway

```python
def central_func(params):
    lookup_dict = {'position': ['behavior', pynwb.behavior.position], 'other': ['field', type]}
    if param in lookup_dict:
        field, type = lookup_dict[param]
    else:
        field, type = default_field, default_type
        
     ```